.editorconfig

```diff
@@ -10,8 +10,4 @@ max_line_length = 100
 indent_size = 2
 
 [*.md]
-trim_trailing_whitespace = false
-
-[*.{js,ts}]
-block_comment_start = /*
-block_comment_end = */
```

Comment:
Why did you remove these rules?

Reply:
This change is the commit for revert merge